graph/types/post.graphqls

```diff
@@ -7,4 +7,6 @@ type Post {
     participant: Participant!
     createdAt: String!
     updatedAt: String!
+    quotedPostID: ID
+    quotedPost: Post
```

Comment:
By virtue of this setup, the quotedPost could have a quotedPost, etc. Should we have a different type here that doesn't have a `quotedPost` field? I guess the backend could always just not send it down which would be equivalent.